pkg/aks/components/AksNodePool.vue

```diff
@@ -334,24 +334,24 @@ export default defineComponent({
       <template v-if="pool.enableAutoScaling">
         <div class="col span-3">
           <LabeledInput
-            v-model.number="pool.minCount"
+            v-model:value.number="pool.minCount"
```

Comment:
I think there are many more instances of this bug in the dashboard. I'm investigating and writing up an issue about it.

Reply:
https://github.com/rancher/dashboard/issues/11856